Week3/homework/js-exercises/calcTotalCost.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+function calculateTotalPrice(receipt) {
+    let totalPrice = 0;
+    for (const propt in receipt) {
+        totalPrice += receipt[propt];
+    }
+    return totalPrice;
+};
+const cartForParty = {
```

Comment:
Very well written code. 👏 

However, I would like to introduce to you 2 new methods which you are going to learn perhaps in JS2. One is called `Object.keys` which you can use currently instead of a `for-in` loop. With modern JavaScript, it is better to make use of `Object.keys` instead of `for-in` loop. You are going to learn about the difference in JS3 when you reach inheritance.

Also, this is a perfect use case of another method in JavaScript called `reduce` which takes in an array and reduces all the elements to a single value which in this case is our totalPrice. You will learn about this method in JS2.